README.md

```diff
@@ -1,258 +1,45 @@
-...
-$ dvc repro
-Stage 'count-words' didn't change, skipping
-Data and pipelines are up to date.
+2. Create and activate your environment, rename the environment if you already have an existing one and would like to preserve it. If you want a fresh start, `conda deactivate` then `conda remove -n inatator`. Then run:
 ```
-
-
-You can see the stages in the DAG by running `dvc dag`, in our case it's just a single step called `count-words`:
+conda create -n inatator python=3.10
+conda activate inatator
 ```
-$ dvc dag
-+-------------+
-| count-words |
-+-------------+
-```
-
-## A Note on Notebooks
-We have also included an example Jupyter notebook
 
-Jupyter notebooks are useful tools for exploratory data analysis, prototyping baseline models and creating visualizations. However, they are _not_ an acceptable way to hand-off code for others to reproduce. Have you ever tried to run someone else's notebook, only to find out a cell was deleted, and you have no idea what it was supposed to do?
+3. At the root of the project `pip install -r src/backend/requirements.txt` and `pip install -r src/backend/requirements-dev.txt`.
 
-[Don't put data science notebooks into production](https://martinfowler.com/articles/productize-data-sci-notebooks.html), they are [hard to test, version, parametrize and keep track of state](https://www.reddit.com/r/datascience/comments/ezh50g/jupyter_notebooks_in_productionno_just_no/).
+## :penguin: Run the Application
+Open two terminals. You will need to run the server first, then application frontend.
+1. ***Run the backend (server):*** To start the server, navigate to project root, run `uvicorn src.backend.app.main:app --reload`, to make sure it is working go to `http://localhost:8000/hello/` in your browser.
+2. ***Run the frontend (application):*** To start the application, navigate to `src/frontend/` you see there are js files, you are at the right spot. Now run `npm start`, it will start running the app and should automatically open to page in your browser.
 
-There _are_ [companies that use notebooks in production architecture](https://blog.goodaudience.com/inside-netflixs-notebook-driven-architecture-aedded32145e), but they have entire Devops organizations to help configure deployment and _still_ use workflow tools like [papermill](https://papermill.readthedocs.io/en/latest/) and Airflow to parametrize notebook dependencies. Unless you are willing to put in the effort to parametrize your notebooks in pipeline workflows, don't use them when stability and reproducibility matter.
+### Running applications with Docker
+1. Install Docker if you haven't already
```

Comment:
Not enough to have it installed, it also needs to be running. Add to the instruction '.. and open the app' or something.